runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -79,8 +82,12 @@
 	private static final int TRAWLER_SHIP_REGION_NORMAL = 7499;
 	private static final int TRAWLER_SHIP_REGION_SINKING = 8011;
 
+	private static final int fishingTrawlerTimeLimit = 614;
+
 	private static final int TRAWLER_ACTIVITY_THRESHOLD = Math.round(0.15f * 255);
 
+	private static Instant endTime = Instant.now().plusSeconds(fishingTrawlerTimeLimit);
```

Comment:
this seems wrong, it should not be static. Also, you probably just need startTime and then manually add the trawler duration to it, so change this to startTime. and initialize it as null. Also reset it on shutDown

Reply:
I've changed endTime to startTime and initalised it as null. When you say "reset on shutdown", is shutdown an event and do you mean setting startTime = null?